streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetricsImpl.java

```diff
@@ -25,34 +25,88 @@
 import org.apache.kafka.common.metrics.stats.Count;
 import org.apache.kafka.common.metrics.stats.Max;
 import org.apache.kafka.common.metrics.stats.Meter;
+import org.apache.kafka.common.metrics.stats.SampledStat;
+import org.apache.kafka.common.metrics.stats.Sum;
 import org.apache.kafka.common.utils.Time;
 import org.apache.kafka.streams.StreamsMetrics;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.List;
-import java.util.ArrayList;
-import java.util.Arrays;
 
-public class StreamsMetricsImpl implements StreamsMetrics {
+public class StreamsMetricsImpl implements StreamsMetrics, InternalStreamsMetrics {
```

Comment:
Just `implements InternalStreamsMetrics` should be sufficient, since `InternalStreamsMetrics` extends `StreamsMetrics`?

Reply:
Ah, yeah, in an earlier pass they were independent interfaces.